v2/clients/http/utils/request.go

```diff
@@ -73,3 +73,47 @@ func PutRequest(
 	}
 	return nil
 }
+
+// Helper method to make the post YAML file request and return the body
+func PostByYamlFileRequest(
```

Comment:
Rename to `PostByFileRequest` since could be used for any file, not just YAML.

Reply:
Renamed.